.github/workflows/qubes-dom0-package.yml

```diff
@@ -100,3 +103,24 @@ jobs:
         with:
           name: qubesos.dom0.fc37-${{ inputs.qubes-component }}-${{ github.sha }}
           path: '*.rpm'
+
+      - name: Construct release's description
+        if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags/')
```

Comment:
Would `github.ref_type` work here as well?

Reply:
`github.ref_type == 'tag'` should be equivalent to the second expression.  I just didn't see that field.